tests/testreactive.py

```diff
@@ -128,6 +128,12 @@ def test_numpy_ufunc(ufunc):
     array = np.ndarray([1, 2, 3])
     assert ufunc(rx(array)).rx.value == ufunc(array)
 
+def test_reactive_empty_construct():
+    i = rx()
+    assert i.rx.value is None
```

Comment:
Curious, did you consider using `Undefined` here?

Reply:
Yes, but we really haven't ever made `Undefined` user facing. It's only really been used an internal sentinel value.